applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -68,6 +70,15 @@ def _SetUpCompressiblePerturbationElement(self, formulation_settings):
         self.element_name = "CompressiblePerturbationPotentialFlowElement"
         self.condition_name = "PotentialWallCondition"
 
+    def _SetUpTransonicPerturbationElement(self, formulation_settings):
+        default_settings = KratosMultiphysics.Parameters(r"""{
+            "element_type": "perturbation_transonic"
+        }""")
+        formulation_settings.ValidateAndAssignDefaults(default_settings)
+
+        self.element_name = "TransonicPerturbationPotentialFlowElement"
```

Comment:
I insist in the naming convention, the element name should clearly state that it is embedded.

Reply:
This is for the body fitted case, I would leave embedded for the elements developed by @marcnunezc to distinguish them better

Reply:
Agree. Now I understand that these embed the wake.